vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -14,11 +14,11 @@ The Hub allows users to:
 - Perform business-level assessment of component readiness. 
 
 An Entando Application can make use of an Entando Hub in several ways:
-* The Local Hub, included in the Entando App Builder, displays a collection of components ready for use. These can be used to compose an application or as a starting point to create new components.	
-* Entando Cloud Hub is the public catalog containing packaged business capabilities and components provided by Entando and its partners throughout the world.
-* An enterprise Entando Hub, developed and curated by Entando clients and partners, can be used to share components within their organization or made available  for public use.
+* The **Local Hub**, included in the Entando App Builder, displays a collection of components ready for use to compose an application or as a starting point to create new implementations.	
```

Comment:
the grammar is a bit off here: collection of components...as a starting point -> collection of components...which comprise (just one example)

"ready for use" struck me as off so i googled -> ready to use. "ready-to-use" also shows up as a modifier, so "...components ready to use for application composition..." or "...ready-to-use components to compose..." (etc.)

"implementations" is a bit vague and imprecise; what you had previously is more accurate, where these components are the starting point to create new components, although i realize we want to avoid that repetition. i actually like what you had before (the 2 sentences, with ready for use -> ready to use) - simple and clear and correct